<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1076</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_9">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QGroupBox" name="groupBox_7">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>30</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Sequence</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="sequenceLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_8">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Configuration</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="1" column="1">
            <widget class="QGroupBox" name="groupBox_10">
             <property name="title">
              <string>TE</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_6">
              <item row="0" column="0">
               <widget class="QDial" name="dial_2"/>
              </item>
             </layout>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QGroupBox" name="groupBox_12">
             <property name="title">
              <string>RF</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_8">
              <item row="0" column="0">
               <widget class="QDial" name="rfDial">
                <property name="maximum">
                 <number>90</number>
                </property>
                <property name="value">
                 <number>90</number>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QGroupBox" name="groupBox_11">
             <property name="title">
              <string>GroupBox</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_7">
              <item row="0" column="0">
               <widget class="QDial" name="dial_4"/>
              </item>
             </layout>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QGroupBox" name="groupBox_9">
             <property name="title">
              <string>TR</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="0" column="0">
               <widget class="QDial" name="dial"/>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>50</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Reconstructed Image</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="1" column="0" colspan="2">
         <widget class="ImageView" name="reconstructedView">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>70</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="reconstDefaultSize">
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>icons/center_gray.png</normaloff>icons/center_gray.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="widget_6" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>20</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
            <horstretch>25</horstretch>
            <verstretch>5</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Weight</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QRadioButton" name="pdRadioButton">
             <property name="text">
              <string>PD</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="t1RadioButton">
             <property name="text">
              <string>T1</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="t2RadioButton">
             <property name="text">
              <string>T2</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QGroupBox" name="groupBox_6">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>30</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="title">
           <string>Marker</string>
          </property>
          <widget class="QLabel" name="label_value">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>269</width>
             <height>24</height>
            </rect>
           </property>
           <property name="text">
            <string>NA</string>
           </property>
          </widget>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>20</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Phantom Size</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QComboBox" name="sizeComboBox">
             <item>
              <property name="text">
               <string>16x16</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>32x32</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>64x64</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="phantomDefaultSize">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>icons/center_gray.png</normaloff>icons/center_gray.png</iconset>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="4">
         <widget class="QGroupBox" name="groupBox_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>40</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Phantom</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="ImageView" name="phantomView">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>30</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0" colspan="4">
         <widget class="QGroupBox" name="groupBox_4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>40</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>K-Space</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="ImageView" name="kspaceView">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>30</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1076</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuImport">
     <property name="title">
      <string>Import</string>
     </property>
     <addaction name="actionPhantom"/>
     <addaction name="actionSequence_2"/>
    </widget>
    <addaction name="menuImport"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>25</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="separator"/>
   <addaction name="actionReconstruct"/>
   <addaction name="actionStop"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionSequence">
   <property name="text">
    <string>Sequence</string>
   </property>
  </action>
  <action name="actionPhantom">
   <property name="text">
    <string>Phantom</string>
   </property>
  </action>
  <action name="actionSequence_2">
   <property name="text">
    <string>Sequence</string>
   </property>
  </action>
  <action name="actionReconstruct">
   <property name="icon">
    <iconset>
     <normaloff>icons/play_gray.png</normaloff>
     <normalon>C:/Users/Mohamed/.designer/backup/icons/play_gray.png</normalon>icons/play_gray.png</iconset>
   </property>
   <property name="text">
    <string>Reconstruct</string>
   </property>
   <property name="toolTip">
    <string>Reconstruct</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset>
     <normaloff>icons/stop_gray.png</normaloff>icons/stop_gray.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="toolTip">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset>
     <normaloff>icons/open-folder.png</normaloff>icons/open-folder.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageView</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
